### PR TITLE
Add gradio UI

### PR DESCRIPTION
**Pull request by WeberJulian**

### CODE REVIEW

File: demo.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import json
+import os
+
+
+SERVER_URL = 'http://localhost:8000'
+OUTPUT = "./demo_outputs"
+cloned_speakers = {}
+
+print("Preparing file structure...")
+if not os.path.exists(OUTPUT):
```
Comment (reuben) [inline demo.py:14]: Better to use `os.makedirs(..., exist_ok=True)`, that way the folder structure is also repaired if it's incomplete.

File: demo.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    os.mkdir(OUTPUT)
```
Comment (reuben) [inline demo.py:15]: Side effects should never run top-level, always be contained inside an `if __name__ == '__main__':` block.

**Review by reuben — APPROVED** [review]
Other than the inline comments it looks good to me, but I don't know much about Gradio.

File: demo.py (hunk 2 above)
Comment (WeberJulian) [reply demo.py:15]: Yeah but I couldn't make it work, here I define STUDIO_SPEAKERS  for example that I use below, if I move it in the if main I get `NameError: name 'STUDIO_SPEAKERS' is not defined`. Maybe we can update in the next PR?

**Pull request merged**